rc/base/ctags.kak

```diff
@@ -7,29 +7,54 @@ decl str-list ctagsfiles 'tags'
 
 def -params 0..1 \
     -shell-candidates '
-        ( for tags in $(printf %s\\n "${kak_opt_ctagsfiles}" | tr \':\' \'\n\'); do
-              namecache=$(dirname ${tags})/.kak.$(basename ${tags}).namecache
-              if [ -z "$(find ${namecache} -prune -newer ${tags})" ]; then
-                  cat ${tags} | cut -f 1 | uniq > ${namecache}
-              fi
-              cat ${namecache}
-          done )' \
+        {
+        oldIFS="$IFS"
+        IFS=:
```

Comment:
In which case, why do we need oldIFS for ? we know IFS is supposed to be the default one right ? so we can just unset IFS to restore it.